src/pineko/evolve.py

```diff
@@ -149,7 +154,10 @@ def evolve_grid(
     comparison_pdf : None or str
         if given, a comparison table (with / without evolution) will be printed
     """
-    x_grid, _pids, mur2_grid, _muf2_grid = grid.axes()
+    order_mask = pineappl.grid.Order.create_mask(grid.orders(), max_as, max_al)
+    evolve_info = grid.raw.evolve_info(order_mask)
```

Comment:
In general, you should be able to call methods on PineAPPL objects without using `.raw`
```suggestion
    evolve_info = grid.evolve_info(order_mask)
```

If there is any problem with this, please let me know :)